changelog/change_mark_style_for_as_unsafe.md

```diff
@@ -0,0 +1 @@
+* [#10427](https://github.com/rubocop/rubocop/issues/10427): Mark `Style/For` as unsafe auto-correction. ([@issyl0][])
```

Comment:
I think, as a first time contributor, you need to add your name at the bottom of CHANGELOG.md. See the last bullet point on https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md.

Reply:
There is no need to manually update the CHANGELOG.md file as first time contributor name will be inserted automatically :-)

Reply:
That's great! So much automation going on that I haven't kept up with.